OpenRA.Mods.Common/Traits/Player/PlaceBeacon.cs

```diff
@@ -19,8 +19,14 @@ public class PlaceBeaconInfo : ITraitInfo
 		public readonly int Duration = 30 * 25;
 		public readonly string NotificationType = "Sounds";
 		public readonly string Notification = "Beacon";
+
+		[PaletteReference(true)]
 		public readonly string PalettePrefix = "player";
 
+		public readonly string BeaconImage = "beacon";
+		[SequenceReference("BeaconImage")] public readonly string ArrowSprite = "arrow";
+		[SequenceReference("BeaconImage")] public readonly string CircleSprite = "circles";
```

Comment:
Shouldn't these be `ArrowSequence` and `CircleSequence` now? You call them that in the support power traits already.